curriculum/challenges/english/20-upcoming-python/learn-recursion-by-solving-the-tower-of-hanoi-puzzle/64e33a488b4a2717fa22ebf3.md

```diff
@@ -0,0 +1,41 @@
+---
+id: 64e33a488b4a2717fa22ebf3
+title: Step 15
+challengeType: 20
+dashedName: step-15
+---
+
+# --description--
+
+Add another comment before your `print()` call saying `display starting configuration`.
```

Comment:
```suggestion
Add another comment before your `print()` call saying `Display starting configuration`.
```

Reply:
I would suggest starting comments with an uppercase letter, like in the examples from [PEP 8](https://peps.python.org/pep-0008/#inline-comments).